index.tt

```diff
@@ -6,152 +6,353 @@
 
 <link rel="alternate" type="application/rss+xml" title="RSS" href="/news-rss.xml" />
 
-<div class="nixos-org-index">
-  [%# Title for the site, used to keep the document tree rooted at h1. %]
-  <h1>NixOS</h1>
-  
-  <div class="jumbotron-two"><div>
-    <div class="intro-nix">
-      <h2>What is Nix?</h2>
-    
-      <p>
-      Nix is a powerful package manager for Linux and other Unix systems
-      that makes package management reliable and reproducible.  It
-      provides atomic upgrades and rollbacks, side-by-side installation
-      of multiple versions of a package, multi-user package management
-      and easy setup of build environments.
-      </p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nix/">
-          <i class="fa fa-chevron-right"></i> Learn more about Nix
-        </a>
+<div class="row-fluid">
+
+  <div class="span6">
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      NixOS package manager can be used independently on any POSIX compliant
```

Comment:
This should read something like "Nix is a package manager for Linux and macOS that allows you to reliably share...". Although the word "share" is a bit vague...

Reply:
And make the words Nix and NixOS bold, like in ldleworks example. I also like the sentences that short, just visually.

![2020-04-29-20-50-30](https://user-images.githubusercontent.com/91113/80649826-0ea83c00-8a73-11ea-8d0a-ecd7a07b7612.png)

Reply:
I bolded "Nix" and "NixOS" but I kept the current text that is on nixos.org. We can/should update it in follow up PRs.